frontend/src/components/credentials/actions/CreateCredential.tsx

```diff
@@ -203,7 +203,7 @@ export default function CreateCredential({
               for information on setting up this connector.
             </p>
           )}
-          <CardSection className="w-full  !border-0 mt-4 flex flex-col gap-y-6">
+          <CardSection className="!border-0 mt-4 flex flex-col gap-y-6">
```

Comment:
remove ! typo

Reply:
! is to override other style (if any), mean remove any border style
present in original code, why typo?